rust/crd/Cargo.toml

```diff
@@ -5,20 +5,19 @@ edition = "2021"
 license = "OSL-3.0"
 name = "stackable-trino-crd"
 repository = "https://github.com/stackabletech/trino-operator"
-version = "0.0.0-dev"
 publish = false
+version.workspace = true
```

Comment:
This will probably break release scripts CC @adwk67 @razvan

Reply:
This is fine. The release scripts don't change dependencies.

Reply:
Looking at https://github.com/stackabletech/stackable-utils/blob/main/release/cargo-version.py are you sure it works? ;) Just want to be save

Reply:
I think I would suggest to revert the version consolidation change as it does not seem worth it to me. WDYT?
```
➜  trino-operator git:(bump/kube) /home/sbernauer/stackabletech/stackable-utils/release/cargo-version.py --set 0.0.0-pr123
Traceback (most recent call last):
  File "/home/sbernauer/stackabletech/stackable-utils/release/cargo-version.py", line 172, in <module>
    new.save(old)
  File "/home/sbernauer/stackabletech/stackable-utils/release/cargo-version.py", line 138, in save
    self.crates[cn].save(previous.crates[cn])
  File "/home/sbernauer/stackabletech/stackable-utils/release/cargo-version.py", line 83, in save
    line = line.replace(previous.version, self.version)
TypeError: replace() argument 1 must be str, not dict
```

Reply:
Ah, awesome! https://github.com/stackabletech/trino-operator/blob/f0243a415e1d25c7cf60a2da55cf61a8d53f237d/.github/workflows/build.yml#L317
I can confirm `cargo set-version --offline --workspace 0.0.0-pr123` still works :+1: